libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -245,7 +245,25 @@ SymbolicTupleVariable::SymbolicTupleVariable(
 	solAssert(m_sort->kind == Kind::Tuple, "");
 }
 
-vector<SortPointer> const& SymbolicTupleVariable::components()
+smtutil::Expression SymbolicTupleVariable::currentValue(frontend::TypePointer const& _targetType) const
+{
+	if (!_targetType || sort() == smtSort(*_targetType))
+		return SymbolicVariable::currentValue();
+
+	auto thisTuple = dynamic_pointer_cast<TupleSort>(sort());
+	auto otherTuple = dynamic_pointer_cast<TupleSort>(smtSort(*_targetType));
```

Comment:
@hrkrshnn @chriseth 
This used to be
`auto const& otherTuple = dynamic_cast<TupleSort const&>(*smtSort(*_targetType));`
I guess the OSX libc++ doesn't know how to get references of temporaries (the way the proper libc++ does), it was giving an empty object here, so the assertion on the size failed with 2 vs 0. This seemed to work inside CI.